### ISSUE TITLE
Only one subplot drawn when using gr.beginprint() (Python)

### ISSUE THREAD

**Issue by Modiug** [sciapp/gr#87]
Version is 1.10.0 (Python)

The following script will open a "GKS QtTerm" correctly showing two subplots. However the gr_test.png created with gr.beginprint() only shows the lower subplot.

```
import gr
from gr.pygr import mlab

gr.beginprint('gr_test.png')

mlab.figure()
mlab.subplot(2,1,1)
mlab.plot(range(100), lambda x: x**2)
mlab.subplot(2,1,2)
mlab.plot(range(100), lambda x: x**3)

gr.endprint()
```

Regards
Guido

**Comment by FlorianRhiem** [comment]
Hello Guido, thanks for reporting this. It seems to be a bug in the cairoplugin caused by setting the workstation viewport repeatedly. As a quick workaround, you can export a pdf instead of a png, which should work correctly.

**Comment by FlorianRhiem** [comment]
This should be fixed by commit 01560eacde287732f6b29a03b5fa90cc17953985.

**Comment by Modiug** [comment]
Hi Florian, thanks, but I was actually using the gr.beginprint() mechanism to write to memory for integration with tkinter/PySimpleGUI. Will wait for the next release then (not urgent).

**Comment by FlorianRhiem** [comment]
Ah, the memory output also uses the cairoplugin, so the fix should work as well.

**Comment by FlorianRhiem** [comment]
The fix was released as part of gr 0.41.0 / python-gr 1.11.0.
